### PR TITLE
added VehicleFilterStructure and VehicleFilter

### PR DESCRIPTION
**Pull request by ue71603**
Added VehicleFilter. It is currently a mix between Vehicles and TRAIN NUMBER filter.

### CODE REVIEW

File: OJP/OJP_Common.xsd [hunk 1]
```diff
@@ -188,6 +188,28 @@
...
+		</xs:annotation>
+		<xs:sequence>
+			<xs:element name="Exclude" type="xs:boolean" default="true" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Whether Vehicle and TRAIN NUMBERS in list are to include or exclude from search. Default is exclude.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="VehicleRef" type="siri:VehicleRefStructure" minOccurs="0" maxOccurs="unbounded">
+				<xs:annotation>
+					<xs:documentation>Reference to transport operator</xs:documentation>
```
Comment (trurlurl) [inline OJP/OJP_Common.xsd:203]: operator?

Comment (Aurige) [reply OJP/OJP_Common.xsd:203]: isn't there already is already an "OperatorFilter" (type="OperatorFilterStructure") ?

File: OJP/OJP_Common.xsd [hunk 2]
```diff
@@ -188,6 +188,28 @@
...
 		</xs:sequence>
 	</xs:group>
+	<xs:complexType name="VehicleFilterStructure">
+		<xs:annotation>
+			<xs:documentation>Filter for Vehicles</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:element name="Exclude" type="xs:boolean" default="true" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Whether Vehicle and TRAIN NUMBERS in list are to include or exclude from search. Default is exclude.</xs:documentation>
```
Comment (sgrossberndt) [inline OJP/OJP_Common.xsd:198]: Whether to include or exclude given VehicleRefs and TRAIN NUMBERS in the list from the search. Default is exclude.

File: OJP/OJP_Common.xsd (hunk 1 above)
Comment (sgrossberndt) [inline OJP/OJP_Common.xsd:203]: Unadapted copy-paste documentation of `OperatorRef`

**Review by sgrossberndt — CHANGES_REQUESTED** [review]

File: OJP/OJP_Common.xsd (hunk 1 above)
Comment (ue71603) [reply OJP/OJP_Common.xsd]: copy/paste error corrected

File: OJP/OJP_Common.xsd (hunk 2 above)
Comment (ue71603) [reply OJP/OJP_Common.xsd]: copied your text

File: OJP/OJP_Common.xsd (hunk 1 above)
Comment (ue71603) [reply OJP/OJP_Common.xsd]: updated

**Review by sgrossberndt — APPROVED** [review]

**Review by trurlurl — APPROVED** [review]

**Comment by trurlurl** [comment]
Documentation done.

**Review by skinkie — APPROVED** [review]

**Review by Aurige — APPROVED** [review]

**Review by sgrossberndt — APPROVED** [review]

**Pull request merged**